collector/VERSION

```diff
@@ -1 +1 @@
-v0.1.0
+v0.58.0
```

Comment:
ADOT collector Lambda layer version should align with ADOT collect, not opentelemetry collector.
Does ADOT collector version change to 0.58.0 now?

Reply:
The ADOT collector is aligned with v0.58.0 version of opentelemetry-collector. The ADOT collector is currently at `v0.21.0`. So sure, the Version needs to v0.21.0 then 🙂

Reply:
@wangzlei  can you please confirm if that sounds good to you 🙂

Reply:
I think following the OTel Collector (not the ADOT Collector) makes sense because this repo is shared by everyone using open source. Let's bump it then.